### PR TITLE
DE44044 - Remove tab trap

### PR DESCRIPTION
**Pull request by devpow112**
This removes the tab stop on entering the override element and re-adds it on exiting. Also removed uses of `var` in the code I was changing.

### CODE REVIEW

**Comment by devpow112** [comment]
Not sure if this will work. Still trying to test on our janky Mac set-up.

File: d2l-rubric-editable-score.js [hunk 1]
```diff
@@ -368,31 +370,52 @@ Polymer({
 		this._scoreModified = true;
 	},
 
+	_focusHandler: function(e) {
+		if (!e || !e.target) {
+			return;
+		}
```
Comment (semibran) [inline d2l-rubric-editable-score.js:376]: I don't think it's possible for the event to come up undefined in this case

Comment (devpow112) [reply d2l-rubric-editable-score.js:376]: Fair, I'm just used to null checking everything. I can remove it if we want.

Comment (semibran) [reply d2l-rubric-editable-score.js]: the things i'd give to setup optional chaining in this repo.......

**Review by semibran — APPROVED** [review]

File: d2l-rubric-editable-score.js (hunk 1 above)
Comment (devpow112) [reply d2l-rubric-editable-score.js]: Lol someday...

**Pull request merged**